### PR TITLE
chore: replace just-repo-utils with workspace-tools

### PR DESCRIPTION
**Pull request by tido64**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Removes `just-repo-utils` in favour of `workspace-tools`.

### Verification

- Verified that test apps work
- Tests pass

### Pull request checklist

This PR has considered (when applicable):
- [x] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/android/metro.config.js [hunk 1]
```diff
@@ -5,15 +5,15 @@
  * @format
  */
 
-const {getWatchFolders} = require('@uifabricshared/build-native');
+const {defaultWatchFolders} = require('@rnx-kit/metro-config');
```
Comment (Saadnajmi) [inline apps/android/metro.config.js:8]: less references to "@ uifabricshared" makes me happy :D

File: package.json [hunk 2]
```diff
@@ -26,7 +26,6 @@
     "checkchange": "beachball check --changehint \"Run 'yarn change' to generate a change file\"",
     "depcheck": "rnx-dep-check && lage depcheck",
     "lint": "lage lint",
-    "postinstall": "node ./scripts/postinstall.js",
```
Comment (Saadnajmi) [inline package.json:29]: Why is this removed?

**Review by Saadnajmi — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (tido64) [reply package.json:29]: The script is deleted 😄 
image

Comment (Saadnajmi) [reply package.json:29]: Ah, it's functionality moved upstream. Thanks!
